server/container_execsync.go

```diff
@@ -16,13 +15,13 @@ func (s *Server) ExecSync(ctx context.Context, req *pb.ExecSyncRequest) (*pb.Exe
 		return nil, status.Errorf(codes.NotFound, "could not find container %q: %v", req.ContainerId, err)
 	}
 
-	if !c.IsAlive() {
-		return nil, fmt.Errorf("container is not created or running")
+	if err := c.IsAlive(); err != nil {
+		return nil, status.Errorf(codes.NotFound, "container is not created or running: %v", err)
```

Comment:
Can you include the container id as well?

Reply:
it's included in the return value of IsAlive()

Reply:
should I move that up a level?

Reply:
No, it's fine!